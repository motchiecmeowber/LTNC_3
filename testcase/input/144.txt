INSERT x number
BEGIN
INSERT y string
BEGIN
INSERT x string
END
PRINT
END