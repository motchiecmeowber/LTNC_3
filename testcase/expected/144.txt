success
success
success
x//0 y//1